### ISSUE TITLE
idea for simple modules Stereo merge and stereo split

### ISSUE THREAD

**Issue by matthleg** [stoermelder/vcvrack-packone#222]
hello,
Thanks for making polyphonic the intermix,
it would be nice now to have two small module to merge and split 8 stereo signals as in the image.
Please think about it.
Have a nice Christmas time.
Matthieu
![stmg](https://user-images.githubusercontent.com/36376831/102715609-e9549080-42d6-11eb-98d0-09fd3316b911.jpg)
![stsp](https://user-images.githubusercontent.com/36376831/102715612-e9ed2700-42d6-11eb-825d-2bde084ab39e.jpg)

**Comment by stoermelder** [comment]
I don't think it is worth another module, there a several split modules available, also a stereo merge/split:
https://library.vcvrack.com/LindenbergResearch/PolyStereo
